infrastructure/modules/armonik/ingress-cert.tf

```diff
@@ -131,16 +181,30 @@ resource "local_sensitive_file" "ingress_ca" {
   file_permission = "0600"
 }
 
+resource "local_sensitive_file" "ingress_client_ca" {
+  count           = length(tls_self_signed_cert.client_root_ingress)
+  content         = tls_self_signed_cert.client_root_ingress.0.cert_pem
+  filename        = "${path.root}/generated/certificates/ingress/client_ca.crt"
+  file_permission = "0600"
+}
+
 resource "local_sensitive_file" "ingress_client_crt" {
   count           = length(tls_locally_signed_cert.ingress_client_certificate)
-  content         = tls_locally_signed_cert.ingress_client_certificate.0.cert_pem
-  filename        = "${path.root}/generated/certificates/ingress/client.crt"
+  content         = tls_locally_signed_cert.ingress_client_certificate[count.index].cert_pem
+  filename        = length(tls_locally_signed_cert.ingress_client_certificate) > 1 ? "${path.root}/generated/certificates/ingress/client.${local.ingress_generated_cert.names[count.index]}.crt" : "${path.root}/generated/certificates/ingress/client.crt"
   file_permission = "0600"
 }
 
 resource "local_sensitive_file" "ingress_client_key" {
   count           = length(tls_private_key.ingress_client_private_key)
-  content         = tls_private_key.ingress_client_private_key.0.private_key_pem
-  filename        = "${path.root}/generated/certificates/ingress/client.key"
+  content         = tls_private_key.ingress_client_private_key[count.index].private_key_pem
+  filename        = length(tls_private_key.ingress_client_private_key) > 1 ? "${path.root}/generated/certificates/ingress/client.${local.ingress_generated_cert.names[count.index]}.key" : "${path.root}/generated/certificates/ingress/client.key"
+  file_permission = "0600"
+}
+
+resource "local_sensitive_file" "ingress_client_p12" {
+  count           = length(pkcs12_from_pem.ingress_client_pkcs12)
```

Comment:
You should use a `for_each` with the `pkcs12_from_pem.ingress_client_pkcs12` map